tests/generic_config_updater/test_lo_interface.py

```diff
@@ -28,7 +28,7 @@
 REPLACE_IPV6 = "FC00:1::210/128"
 
 pytestmark = [
-    pytest.mark.topology('t0', 'm0', 'mx'),
+    pytest.mark.topology('t0', 'm0', 'mx'), # M1/M2/M3: ?
```

Comment:
need revise what is the purpose of this scenario

Reply:
Had a glance on the code. It's for testing loopback configuration.
I'm checking with Jingwen if this can be enable on T1/M1/M2/M3 too.

Reply:
Confirmed this case can enable on T1, so do M1/M2/M3.